apps/dashboard/src/components/workflow-editor/url-input.tsx

```diff
@@ -82,9 +75,7 @@ export const URLInput = ({
           </InputRoot>
         </div>
       </div>
-      <FormMessagePure error={error ? String(error.message) : undefined}>
-        {withHint && 'Type {{ for variables'}
-      </FormMessagePure>
+      <FormField control={control} name={urlKey} render={() => <FormMessage />} />
```

Comment:
Print the URL error under the composite URLInput. See the following examples:

<img width="608" alt="Screenshot 2025-03-28 at 00 30 42" src="https://github.com/user-attachments/assets/3ed94f55-8026-47d6-b2ea-a1675ef8542a" />

<img width="598" alt="Screenshot 2025-03-28 at 00 30 59" src="https://github.com/user-attachments/assets/35b7d501-46dc-4bcf-803c-3b3b1653492c" />